en/docs/insights/derive-insights.md

```diff
@@ -0,0 +1,364 @@
+
+![Device statistics](../assets/img/insights/devices-page-full.png){.cInlineImage-full}
+
+To filter the information displayed on this page by API, select the required API(s) in the **API** field. **All** is selected by default.
+
+The information displayed is as follows:
+
+- **Top Platforms**
+
+    This chart shows the Operating System Platform wise breakdown of the API clients. The pie chart only shows platforms with a significant amount of requests. Other platforms are grouped under the **Other** category.
+
+    ![Top platforms](../assets/img/insights/top-platforms.png){.cInlineImage-half}
+
+- **Top User Agents**
+
+    This chart shows the breakdown of the API clients based on user-agent. The pie chart only shows user agents with a significant amount of requests. Other platforms are grouped under the **Other** category.
+
+    ![Top user agents](../assets/img/insights/top-user-agents.png){.cInlineImage-half}
+    
+### Alerts
+
+This page shows information related to business alerts issued by Choreo for your currently selected environment and organization. You can use this page as a health monitoring dashboard and make it visible to your project team. Using this page you can drill down on each alert and discover possible anomalies in your published APIs. See the topics given below to get details of the available widgets and understand how to use them to the best effect.
+
+**Prerequisites**
+
+To use this page, be sure to configure alerts and have API invocations that trigger alerts. For instructions, see [Alert Configuration]
+
+!!! Tip
+    Alerts are retained only for a week.
+
+- **Alert Summary**
+
+    This table lists down each alert generated during the selected time interval. The message of the alert can be viewed by expanding the arrow icon on the details column. The dropdowns can be used to filter alerts based on type and API.
+
+    ![Alert summary](../assets/img/insights/alert-summary.png){.cInlineImage-full}
+
+- **Top APIs by Alert**
+
+    This shows a pie chart and a table to visualize the alert distribution during the selected time interval.
+    
+    ![Top APIs by alert](../assets/img/insights/top-apis-by-alert-count.png){.cInlineImage-full}    
+
+### Reports
+
+This page allows you to download monthly usage reports for your system. There are pre-configured reports with system-wide statistics, and a custom report generator to generate reports based on a subset of APIs/applications.
+
+![Report page](../assets/img/insights/report-page-full.png){.cInlineImage-full}
+
+#### Download custom reports
+
+This widget allows you to generate a custom report with available stats and download it.
+
+![Custom report download](../assets/img/insights/custom-report.png){.cInlineImage-full}
+
+To generate a custom report, follow the steps below:
+
+1. In the **API** field, select one or more APIs for which you want to generate the report. If required, you can select all the APIs.
```

Comment:
The suggestion involves repeating the word "select". Shall we leave it this way because the "All" checkbox is quite visible and it is clear that the user needs to select that to select all the APIs?

Reply:
I think its better to explicitly mention the value.

Reply:
Changed.